scripts/python/repos.py

```diff
@@ -717,6 +717,7 @@ def sync(self, pkg_list, alt_url=None, py_ver=27):
             pkg_list (str): list of packages separated by space(s). Packages can
                 include versions. ie Keras==2.0.5
         """
+        _python = 'python2.7' if py_ver == 27 else 'python'
```

Comment:
I don't think this works. Since you don't know if the system has pip installed for python2.7 Actually that is exactly the issue I am facing when I moved to latest change. I believe the reason why it was left before with pup python was because one knows that pip is installed in that virtual environment.